e2e/simple/WORKSPACE

```diff
@@ -0,0 +1,23 @@
+# Override http_archive for local testing
```

Comment:
our current setup in other repos has an examples/ folder which is not a nested workspace. I think we should stay consistent (if we're going to change, we should change all of them)

Reply:
Oh, I didn't mean this to be an example. I should rename it. I wanted to assert jasmine_test is usable from another workspace.

Reply:
I'll add an example folder as well.